src/main/java/kitchenpos/domain/OrderTable.java

```diff
@@ -1,60 +1,114 @@
 package kitchenpos.domain;
 
+import javax.persistence.Column;
+import javax.persistence.Embedded;
+import javax.persistence.Entity;
+import javax.persistence.ForeignKey;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import kitchenpos.domain.common.NumberOfGuests;
+
+@Entity
+@Table(name = "order_table")
 public class OrderTable {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
-    private Long tableGroupId;
-    private int numberOfGuests;
+
+    @ManyToOne
+    @JoinColumn(name = "table_group_id", foreignKey = @ForeignKey(name = "fk_order_table_table_group"))
+    private TableGroup tableGroup;
+
+    @Embedded
+    private NumberOfGuests numberOfGuests;
+
+    @Column(nullable = false)
     private boolean empty;
 
-    public OrderTable() {
+    protected OrderTable() {
     }
 
     public OrderTable(Long id, boolean empty) {
         this.id = id;
         this.empty = empty;
     }
 
-    public OrderTable(Long id, Long tableGroupId, boolean empty) {
+    public OrderTable(Long id, TableGroup tableGroup, boolean empty) {
         this.id = id;
-        this.tableGroupId = tableGroupId;
+        this.tableGroup = tableGroup;
+        this.empty = empty;
+    }
+
+    public OrderTable(TableGroup tableGroup, int numberOfGuests, boolean empty) {
+        this.tableGroup = tableGroup;
+        this.numberOfGuests = new NumberOfGuests(numberOfGuests);
         this.empty = empty;
     }
 
     public OrderTable(Long id, int numberOfGuests, boolean empty) {
         this.id = id;
-        this.numberOfGuests = numberOfGuests;
+        this.numberOfGuests = new NumberOfGuests(numberOfGuests);
         this.empty = empty;
     }
 
-    public Long getId() {
-        return id;
+    public OrderTable(int numberOfGuests, boolean empty) {
+        this.numberOfGuests = new NumberOfGuests(numberOfGuests);
+        this.empty = empty;
     }
 
-    public void setId(final Long id) {
-        this.id = id;
+    public void attachToTableGroup(TableGroup tableGroup) {
+        this.tableGroup = tableGroup;
     }
 
-    public Long getTableGroupId() {
-        return tableGroupId;
+    public void updateNumberOfGuests(final int numberOfGuests) {
+        if (!isEmpty()) {
+            throw new IllegalArgumentException("손님수를 변경 할 수 없습니다. 빈 테이블이 아닙니다.");
+        }
+        this.numberOfGuests = new NumberOfGuests(numberOfGuests);
     }
 
-    public void setTableGroupId(final Long tableGroupId) {
-        this.tableGroupId = tableGroupId;
+    public void clear(Order order) {
+        if (this.empty) return;
```

Comment:
```suggestion
        if (this.empty) {
            return;
        }
```

자바에서는 한 라인도 줄바꿈을 해주는 것이 관례입니다 :)